examples/getstarted/config/functions/bootstrap.js

```diff
@@ -8,6 +8,4 @@
  * run jobs, or perform some special logic.
  */
 
-module.exports = cb => {
-  cb();
-};
+module.exports = () => {};
```

Comment:
should we add `async` here ?

Reply:
It doesn't have to be async the function can just return a promise.